app/views/users/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<%
+=begin%>
+ <p id=”notice”><%= notice %></p> 
+<%
+=end%>
+<div class="container">
+  <h1><%= current_user.name %></h1>
+  <button><%= link_to 'Create Event', new_event_path %></button>
+
+<ul>
+  <% if current_user.events.empty? %>
+    <p>User has no events created</p> 
+  <% else %>
+  <h3 class = "card-title"> Past Events</h3>
+    <% current_user.events.each do |event| %>
+
+      <% if event.date < Time.now %>
+        <li><%= link_to event.name, event %></li>
+        <p class = "card-text ">Description: <%= event.description %> </p>
+      <% elsif current_user.events.empty? %>
+        <p> No past events found </p>
+      <% end %>
```

Comment:
Having `@past_events = current_user.attended_events.previous` in your user `#show` method will save you the hassle of having logic in your views. You will only need to loop through `@past_events`